fcrepo-configs/src/main/java/org/fcrepo/config/OcflPropsConfig.java

```diff
@@ -121,6 +121,12 @@
     @Value("${fcrepo.ocfl.unsafe.write.enabled:false}")
     private boolean unsafeWriteEnabled;
 
+    @Value("${fcrepo.ocfl.id_map_cache.size:1024}")
```

Comment:
I find it's often helpful to include the unit in things like this. eg `fcrepo.ocfl.id_map_cache.size.bytes` and `fcrepo.ocfl.id_map_cache.timeout.minutes`. Or something along those lines.

We're planning on adding more db caches, yeah? Like for containment? I wonder if they should all be grouped along the lines of `fcrepo.db.cache.ocfl.size...` and `fcrepo.db.cache.containment.size...`? Or perhaps `fcrepo.cache.db.ocfl.size...`

Reply:
I did think about that and went back and forth. We also have a cache in the OcflPersistentSessionStorage that could be configurable so maybe `fcrepo.cache....` to keep them together.

As for the `size`, it is the [maximum number of elements](https://github.com/ben-manes/caffeine/blob/e1e0c84d860dc0646c5599c15fa853cefeb53c98/caffeine/src/main/java/com/github/benmanes/caffeine/cache/Caffeine.java#L344-L348). Soooooo `fcrepo.cache.db.ocfl.size.elements` or maybe `fcrepo.cache.db.ocfl.size.entries`?

Reply:
Ah, right! It's the number of entries. :)

I like `fcrepo.cache...` and either of the elements or entries options.